subjects/mobile-dev/map-markers/audit/README.md

```diff
@@ -1,35 +1,37 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
-###### Does the application run without crashes?
+###### Does the app run without crashing?
 
-###### Does the application have Bottom tabbar?
+###### Does the app have `Bottom TabBar`?
 
-###### Does the application show google maps?
+###### Does the app show google maps on the `Map` screen?
 
-###### Does the application show list of favorite places?
+###### Does the app show the list of favorite places on the `Favorite Places` screen?
 
-###### Does the application show list of favorite places after reopening application?
+###### Does the app show the list of favorite places after closing and reopening the app?
 
-###### Does the application contain an information page?
+###### Can you delete a favorite place from your list?
 
-###### Does map show markers with favorite places?
+###### If the favorite place is deleted, does the app remove it from the map?
 
-###### When map is tapped, does dialog window show 2 textfields: name and description?
+###### Does the app contain an information page on the `Info` screen?
```

Comment:
Small differences between name of the page asked to be created in the subject (`InfoPage`)